.github/workflows/merge-pr.yml

```diff
@@ -94,6 +94,10 @@ jobs:
     steps:
       - name: Check Out Repo
         uses: actions/checkout@v3
+      - name: Install Required Packages
+        run: |
+          sudo apt-get update
+          sudo apt install -y protobuf-compiler
       - name: Install Rust Toolchain
         uses: actions-rs/toolchain@v1
```

Comment:
Substrate needs protoc library this pretty much became a requirement starting .3